### PR TITLE
feat(ng6): initial migration to ng-cli6

### PR DESCRIPTION
**Pull request by corinnekrych**
ready to be merged.

todo list:

- [x] migrate editor library to ng6  and use ng-cli6 
- [x] CI/CD to run unit test for library
- [x] migrate demo app to ng6  and use ng-cli6 
- [x] support env variable for demo app. See [workaround](https://github.com/fabric8-analytics/fabric8-analytics-dependency-editor/pull/120/files#diff-7bd4a925c695c2eb0eced3872b9b965fR161)
- [x] update README instructions for new dev environment 
- [x] test demo app and make sure it works as intended
- [x] make sure npm release content all required fiele
- [ ] release version `v1.0.0` with semantic-release

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -4,60 +4,62 @@ image:https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic%20rel
...
+npm run build
+----
+to build and run both demo and lib.
 
-=== Fresh run
+=== Running the demo
 
-If you trying to refresh your installation, you need to run:
+==== Set-up env
+To target PROD env in your demo app, go to `src/environments/environment.ts`
```
Comment (invincibleJai) [inline README.adoc]: should this be `src/environments/environment.prod.ts`

**Comment by corinnekrych** [comment]
@invincibleJai 
I've tested the `npm run build` with ngx-launcher. see this commit: https://github.com/fabric8-launcher/ngx-launcher/pull/360/commits/bd817eb87d4953c27e381ca8cd56f6323e89eb7c
Are we ok to merge that PR and release?

**Comment by invincibleJai** [comment]
@corinnekrych feel free to merge it :)

**Pull request merged**